File: Bridge/Resources/.generated/system/threading/tasks/synchronizationContextScheduler.js
    Bridge.define("System.Threading.Tasks.SynchronizationContextScheduler", {
        inherits: [System.Threading.Tasks.TaskScheduler],
        props: {
            MaximumConcurrencyLevel: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function (ctx) {
                this.$initialize();
                System.Threading.Tasks.TaskScheduler.ctor.call(this);
            }
        },
        methods: {
            queueTask: function (task) { },
            taskLaunchWrapper: function (obj) { },
            getScheduledTasks: function () {
                throw new System.NotImplementedException();
            },
            tryDequeue: function (task) {
                throw new System.NotImplementedException();
            },
            tryExecuteTaskInline: function (task, taskWasPreviouslyQueued) {
                throw new System.NotImplementedException();
            }
        }
    });

File: Bridge/System/Threading/Tasks/SynchronizationContextScheduler.cs
using Bridge;

namespace System.Threading.Tasks
{
    sealed class SynchronizationContextScheduler : TaskScheduler
    {
        public SynchronizationContextScheduler(SynchronizationContext ctx)
        {
        }

        protected internal override void QueueTask(Task task)
        {
        }

        void TaskLaunchWrapper(object obj)
        {
        }

        protected override Collections.Generic.IEnumerable<Task> GetScheduledTasks()
        {
            throw new NotImplementedException();
        }

        protected internal override bool TryDequeue(Task task)
        {
            throw new NotImplementedException();
        }

        protected override bool TryExecuteTaskInline(Task task, bool taskWasPreviouslyQueued)
        {
            throw new NotImplementedException();
        }

        public override int MaximumConcurrencyLevel
        {
            get
            {
                throw new NotImplementedException();
            }
        }
    }
}